### DISCUSSION TITLE
LSposed is not installed

### DISCUSSION THREAD

**Discussion by 0xsharkboy** [LSPosed/LSPosed#2185] [General]
Hey, I was using LSposed and it worked well. But for some reason it stopped working. When I'm launching the manager it says "LSposed is not installed" but I'm flashed the module for zygisk. I tried to reinstall it deleting the /data/adb/lspd and reflashing the module. I didn't find logs, where can I find them to show you ?
I really need the Xposed framework for my banking apps
Thank you ^^ 

![Screenshot_20221018-140505](https://user-images.githubusercontent.com/58356637/196517173-7f349a38-548e-4f5b-9ecb-1856c047d51c.jpg)
![Screenshot_20221018-140514](https://user-images.githubusercontent.com/58356637/196517179-ad5b59a7-e65b-4d81-8a05-0aa6e9fb65ba.jpg)
![Screenshot_20221018-140535_LSPosed](https://user-images.githubusercontent.com/58356637/196517183-c21406f4-4884-4417-becb-686def77b2e8.jpg)

**Comment by 0xsharkboy** [comment]
I just found this [logs](https://we.tl/t-LjAlvvozi5)

**Comment by mywalkb** [comment]
Try the module described [here](https://github.com/topjohnwu/Magisk/issues/6042#issuecomment-1167132605) and increase log buffer to max size in dev settings, so you can capture something useful for analysis.
